BangCommands/Helper.js

```diff
@@ -104,6 +104,7 @@ var BangCommandHelper =
 		var foundPath = "";
 
 		var paths = [
+            path.resolve("/.smelt/plugins/" + name + ".js"),
```

Comment:
Why was this addition needed? I think the line below should have covered this:

`path.resolve(Paths.LocalDirectory + "/.smelt/plugins/" + name + ".js")`

What am I missing?